tests/integ/test_mxnet_train.py

```diff
@@ -77,6 +77,31 @@ def test_deploy_model(mxnet_training_job, sagemaker_session, mxnet_full_version)
         sagemaker_session.sagemaker_client.describe_model(ModelName=model.name)
         assert 'Could not find model' in str(exception.value)
 
+def test_deploy_model_with_tags(mxnet_training_job, sagemaker_session, mxnet_full_version):
```

Comment:
You need 2 blank lines between test methods.

Reply:
Added in the latest commit, thanks.